Search/BingImageSearch/multiFeatured/src/main/java/com/microsoft/azure/cognitiveservices/search/imagesearch/samples/BingImageSearchSample.java

```diff
@@ -293,7 +293,7 @@ public static void main(String[] args) {
             //=============================================================
             // Authenticate
 
-            final String subscriptionKey = System.getenv("AZURE_BING_SAMPLES_API_KEY");
+            final String subscriptionKey = "bbfb78ad189649248750011e83b7b22d";//System.getenv("AZURE_BING_SAMPLES_API_KEY");
```

Comment:
This is totally wrong, we don't use plain keys in the samples! The key should be read from the env var. Please make sure the corresponding resource was deleted to avoid any unauthorized usage.

Reply:
Ack, can't believe I missed that. Apologies - Resource has been deleted.